bzip2-sys/lib.rs

```diff
@@ -1,8 +1,8 @@
 #![doc(html_root_url = "https://docs.rs/bzip2-sys/0.1")]
```

Comment:
Does `#![no_std]` work?

Reply:
yes, and interestingly then the `extern crate core;` is no longer needed.